src/features/contacts/index.tsx

```diff
@@ -95,26 +109,76 @@ export const Contacts = () => {
     },
   ];
 
+  const getLocalStorageSavedPropertyValue = (storageKey: string, propertyToRetrieve: string) => {
```

Comment:
it makes more sense to use the useLocalStorage hook like in /ContentEdit/content-edit.tsx to reduce possible bugs cause by handmade implementation. See https://www.npmjs.com/package/use-local-storage

Reply:
useLocalStorage hook is now integrated.